utils/synthetics/getPrices.ts

```diff
@@ -60,15 +60,15 @@ type PriceInfo = {
   close: number
 }
 
-export function getStableTokenPrice(tokenSymbol: string) {
-  return {
-    tokenSymbol,
-    high: 1,
-    low: 1,
-    open: 1,
-    close: 1,
-  }
-}
+// export function getStableTokenPrice(tokenSymbol: string) {
+//   return {
+//     tokenSymbol,
+//     high: 1,
+//     low: 1,
+//     open: 1,
+//     close: 1,
+//   }
+// }
```

Comment:
is this change expected? if this function is no longer need maybe better to delete it?